tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+      "outDir": "./dist",
+      "allowJs": true,
+      "target": "es5"
```

Comment:
I think it's safe to go for 6 and beyond

Reply:
This was set by default, will update it. I agree 6 should be fine, here's the node documentation confirming the support: https://nodejs.org/en/docs/es6/

Reply:
fyi, cli requires at least nodejs v8.9.0, which is es6+ I think.

Reply:
Updated